### PR TITLE
Binary tree search fix

### PR DESCRIPTION
**Pull request by mansisce**

### CODE REVIEW

File: src/assets/seed/data-structures/BinarySearchTree.js [hunk 1]
```diff
@@ -126,11 +126,13 @@ class BinarySearchTree {
 			// node only has right child
 			else if (!node.left) {
 				node.value = node.right.value
+				node.left = node.right.right
```
Comment (no-stack-dub-sack) [inline src/assets/seed/data-structures/BinarySearchTree.js]: Hi @mansisce, so sorry for the extremely late reply. It looks like this line is supposed to be:

```
node.left = node.right.left
```

If you get around to fixing this I can merge!

**Comment by no-stack-dub-sack** [comment]
@mansisce also, there is a test suite you can run with `yarn test` which should test all the solutions against the built in tests, however it might not be working properly right now. You can give it a try, but as long as the solution is manually passing the tests in the UI (which it should, I've tested it), let's not worry about it for now. There's no CI which will block it from being merged.

**Comment by mansisce** [comment]
@no-stack-dub-sack I've resolved the issue. Please merge it now. Sorry for late response.

**Comment by mansisce** [comment]
@no-stack-dub-sack please merge this

**Comment by no-stack-dub-sack** [comment]
@mansisce that one took a while but glad we got it fixed, thanks! Sorry for the delayed merge, been busy the last couple of months starting a family :-)

**Comment by no-stack-dub-sack** [comment]
@mansisce changes deployed :+1: :smile:

**Pull request merged**